src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -195,6 +218,16 @@ public void findDescriptor_invalidPrefixInput_parseExceptionThrown() throws Pars
                 new FindDescriptor(ArgumentTokenizer.tokenizeWithoutPreamble(invalidLevelOfEducationTypePrefixInput,
                         PREFIX_LEVEL_OF_EDUCATION)));
 
+        String invalidExperiencePrefixInput1 = " y/800"; // large number
+        assertThrows(ParseException.class, () ->
+                new FindDescriptor(ArgumentTokenizer.tokenizeWithoutPreamble(invalidExperiencePrefixInput1,
+                        PREFIX_EXPERIENCE)));
+
+        String invalidExperiencePrefixInput2 = " y/-11"; // negative number
+        assertThrows(ParseException.class, () ->
+                new FindDescriptor(ArgumentTokenizer.tokenizeWithoutPreamble(invalidExperiencePrefixInput2,
+                        PREFIX_EXPERIENCE)));
+
```

Comment:
Since the purpose of this test case is to check whether an invalid output (any invalid output) will result in a `ParseException` being thrown, I believe only 1 out of this 2 is required to check that it is working! You can choose either one :)

All possible invalid args should already be tested in `ExperienceTest.java` after all.

Reply:
Yep!